edgekv/lib/README.md

```diff
@@ -44,11 +46,12 @@ The edgekv.js file must be included alongside `edgekv_tokens.js` which includes
 	 * @param {string} $0.item item key to get from the EdgeKV
 	 * @param {object} [$0.default_value=null] the default value to return if a 404 response is returned from EdgeKV
 	 * @param {number} [$0.timeout=null] the maximum time, between 1 and 1000 milliseconds, to wait for the response
-	 * @returns {object} if the operation was successful, the json response from the EdgeKV or the default_value on 404
+	 * @param {number} [$0.num_retries_on_timeout=null] the number of times to retry a requests when the sub request times out
+	 * @returns {Promise<object>} if the operation was successful, the json response from the EdgeKV or the default_value on 404
 	 * @throws {object} if the operation was not successful,
 	 * 		an object describing the non-200 and non-404 response from the EdgeKV: {failed, status, body}
 	 */
-	async getJson({ namespace = this.#namespace, group = this.#group, item, default_value = null, timeout = null } = {})
+	async getJson({ namespace = this.#namespace, group = this.#group, item, default_value = null, timeout = null, num_retries_on_timeout = null } = {})
 ### putText
```

Comment:
I do not see num_retries_on_timeout for putJson

Reply:
put and delete do not benefit from retries for timeouts

Reply:
ack